README.md

```diff
@@ -0,0 +1,73 @@
+# Flatpak packaging for Delta Chat Desktop
+
+## Building
+
+If you'd like to locally build this flapak, you'll need both `flatpak`
+and `flatpak-builder` installed.  E.g. on Debian you can run `apt
+install flatpak flatpak-builder` to install these tools.
+
+### flatpak dependencies
+
+You need to install the required flatpak runtime, SDK and electron
+BaseApp.  At the time of this writing they are the 18.08 versions, but
+check the `chat.delta.desktop.yml`'s `runtime-version` and
+`base-version` properties for the current version numbers.
+
+```
+flatpak install flathub \
+    org.freedesktop.Platform//18.08 \
+    org.freedesktop.Sdk//18.08 \
+    io.atom.electron.BaseApp//18.08
+```
+
+
+### Building the application
+
+To simply build the application in a build-directory invoke
+`flatpak-builder` pointing to the manifest:
+```
+flatpak-builder build-dir chat.delta.desktop.yml
+```
+
+To install the local build you can add the `--install` flag.  To
+upload the built application to a repository, which can just be a
+local directory, add the `--repo=repo` flag.
+
+### Uploading to flathub
+
+- signing?
+- upload
```

Comment:
@muelli anything which needs to be added to this section?  As I understand currently any commit here is automatically build and signed by flathub so we don't have to do any management of this?

Should we be signing commit to this repo?  http://docs.flatpak.org/en/latest/flatpak-builder.html#signing seems to suggest so but this seems not enforced?  Also not sure if the way they describe there works for this situation?

Reply:
that signing is part of ostree commits, not git. I see how this can cause confusion and it could do with a more explicit mentioning. OTOH, "git" is nowhere mentioned at all on that page so it might be fair to not assume git being dealt with in first place.

And your assessment is correct, every push to master (or beta, for completeness sake) causes a build with downloadable artefacts. So no management needed.

Reply:
Oh, I see how I read that wrong.  Thanks for explaining.  Maybe I just mention that every commit here results in a new published release.